### ISSUE TITLE
New submission with no contributors - Can't conclude the submission

### ISSUE THREAD

**Issue by defstat** [pkp/pkp-lib#7388]
**Describe the bug**
When trying to insert a new submission, if no contributors added, the submission steps can't be concluded. 

**To Reproduce**
1. Go through the steps of a new submission
2. Remove all contributors from step 3
3. Try to conclude submission in step 4

**What application are you using?**
 OJS, OMP or OPS version 3.4 (main branch)

**Additional information**
`SubmissionSubmitStep4Form.inc.php` supposes that there is a `$primaryAuthor` (for example in `line 76`) without checking.

**Comment by NateWr** [comment]
Ok, I'm ok with this for now but once this is merged can you file an issue with these screenshots and I'll clean up the UI for 3.4.

**Comment by bozana** [comment]
Do we allow the submissions/publications without contributors?
Or what is the exact use case for this?
Publishing a submission without contributors will have a huge impact on all other systems we distribute the metadata to...

**Comment by NateWr** [comment]
For use cases, there's https://github.com/pkp/pkp-lib/issues/5955. We haven't implemented that, but hope to eventually.

In the meantime, I think you're right that this will cause problems. But we should probably block things at the publish step rather than the submission step. If we force it at the submission step, we'll get whatever data an author puts in to get past the system.

**Comment by bozana** [comment]
@NateWr, @defstat, it seems it is possible to publish a submission without contributors. 
Shall this be so for 3.4.? -- If so, then all the other places in the system (e.g. Crossref export) needs to be adapted... If not, a new issue should be created, to check the existence of the contributors at the publish step...

**Comment by NateWr** [comment]
I think we should implement these requirements at the point that they are required. For example, we've discussed several ways that Crossref enforces metadata restrictions. In such cases, we should add validation checks to the Crossref plugin itself. And these should be added at publication rather than submission.

There's a case for more pre-submission validation checks. But I think those need to be context settings rather than on by default.

**Comment by bozana** [comment]
Hmmm... I am not sure I understand what should be done now, for 3.4...
